carbon-design-system/carbon-for-ibm-dotcom
# fix(pagination): fix issues with pagination playground story

### Related Ticket(s)

Closes https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/10882
https://jsw.ibm.com/browse/ADCMS-3821

### Description

Fixes issues with two of the knobs on the Pagination > Playground story.

### Changelog

**Changed**

- Updated Pagination > Playground story to allow testing `page-size` and `items-per-page-text` attributes of `<cds-pagination>` component.

### Testing Instructions

* Open Carbon Web Components Storybook
* Navigate to Pagination > Playground
* Change values for "Items per page text" and "Number of rows per page (page-size)" knobs
* You should see the changes reflected in the rendered output
